### PR TITLE
Lws support

### PR DESCRIPTION
**Pull request by TheCharlatan**

### CODE REVIEW

**Comment by TheCharlatan** [comment]
Requires the lws container to be publishe first after https://github.com/farcaster-project/containers/pull/10 is merged.

**Comment by Lederstrumpf** [comment]
```
---- monero_syncer_address_test stdout ----
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Address already in use', tests/functional.rs:1863:26
```
container's been merged - still throws this.

**Comment by TheCharlatan** [comment]
The image is still not deployed yet: https://github.com/orgs/farcaster-project/packages?repo_name=containers . Not sure why though.

File: src/swapd/runtime.rs [hunk 1]
```diff
@@ -125,7 +125,7 @@ pub fn run(
     let temporal_safety = TemporalSafety {
         cancel_timelock: cancel_timelock.as_u32(),
         punish_timelock: punish_timelock.as_u32(),
-        btc_finality_thr: 1,
+        btc_finality_thr: 0,
```
Comment (Lederstrumpf) [inline src/swapd/runtime.rs:128]: seems to be a debugging shortcut?

**Review by Lederstrumpf — APPROVED** [review]
tACK, finally :D

**Pull request merged**